### DISCUSSION TITLE
Adding 'featured fields' to field view in new UI

### DISCUSSION THREAD

**Discussion by LeahBreedenNyaruka** [nyaruka/rapidpro#4453] [Ideas]
When looking at a contact's page, it may be helpful to have featured fields so users can choose some to appear at the top of the page or at the top of the 'fields' tab for the contact.

**Comment by ericnewcomer** [comment]
Does this mean adding the ability to toggle featured status from the contact page? The featured fields are already shown at the top in a separate list.

**Reply by LeahBreedenNyaruka** [reply]
Oh yeah I see that. Maybe the user doesn't have any featured.
